<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>541</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MSDL Gait Monitoring System</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>walk.png</normaloff>walk.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="main_gb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>841</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPubWorld돋움체_Pro Medium</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>System Watcher</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <widget class="QLabel" name="lb_status">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Status : </string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_status_detail">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>KoPubWorld돋움체_Pro Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>코딩을 하다 죽은자의 온기가 느껴집니다...</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>idle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_device">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>131</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QRadioButton" name="rb_imu1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IMU 1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_imu2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IMU 2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_imu3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IMU 3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_imu4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IMU 4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_imu5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IMU 5</string>
     </property>
    </widget>
    <widget class="QLabel" name="img_imu1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>red_button.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="img_imu2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>red_button.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="img_imu3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>red_button.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="img_imu4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>red_button.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="img_imu5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>red_button.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="img_pp2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>red_button.png</pixmap>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_imu6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PP 1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_pp2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PP 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="img_imu6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>red_button.png</pixmap>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_all">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ALL</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_sensor">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>201</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Sensor</string>
    </property>
    <widget class="QRadioButton" name="rb_accX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>acc X</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_accY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>acc Y</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_accZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>acc Z</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_all_sensor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>90</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ALL</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_holder_d">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_holder_e">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="te_msg_detail">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>461</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="plainText">
     <string>NONE</string>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_status_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Subject : </string>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lb_status_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Date :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_subject_name">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>정승민</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="accelerated">
     <bool>true</bool>
    </property>
    <property name="currentSectionIndex">
     <number>1</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>Bluetooth Connection</string>
    </property>
    <widget class="QPushButton" name="bt_conn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CONNECT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_disconn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DISCONNECT</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>460</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>Data Handle Method</string>
    </property>
    <widget class="QPushButton" name="bt_save">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save as CSV</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_plot">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_display">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Display</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
